src/Razor/test/Microsoft.VisualStudioCode.RazorExtension.Test/HoverAssertions.cs

```diff
@@ -12,8 +11,15 @@ internal static class HoverAssertions
     public static void VerifyContents(this LspHover hover, object expected)
     {
         var markup = hover.Contents.Fourth;
-        Assert.Equal(MarkupKind.PlainText, markup.Kind);
-        AssertEx.EqualOrDiff(expected.ToString(), markup.Value.TrimEnd('\r', '\n'));
+
+        var actual = markup.Value.TrimEnd('\r', '\n');
+        if (markup.Kind == MarkupKind.Markdown)
+        {
+            // Remove any horizontal rules we may have added to separate HTML and Razor content
+            actual = actual.Replace("\n\n---\n\n", string.Empty);
```

Comment:
Instead of removing the line here, consider adding `HorizontalRule()` to the expected output, which can be just `string HorizontalRule() => "\n\n---\n\n"` in the VS Code assertions, and a no-op in the VS assertions.